src/__tests__/rate.test.js

```diff
@@ -176,4 +176,33 @@ describe('rate', () => {
       { mu: 25, sigma: 8.204837030780652 },
     ])
   })
+
+  it('accepts a tau term', () => {
+    expect.assertions(1)
+    const a = rating({ mu: 25, sigma: 3 })
+    const b = rating({ mu: 25, sigma: 3 })
+    const [[winner], [loser]] = rate([[a], [b]], {
+      tau: 0.3,
```

Comment:
Yeah it's interesting that the `0.08333` value is suggested... which is what worries me that I didn't implement this right?

`0.3` is a value I chose that works fairly well for me, it's an amount of "stable-state" movability that I'm happy with. I'll try `0.08333` in a sec and add another reply.

Reply:
Here are the results of my script with tau = `0.08333` or `sigma / 100`

![image](https://user-images.githubusercontent.com/1534726/158492467-c2fc2ef4-4882-4f98-9e4b-a9cb9ec7ea3a.png)
![image](https://user-images.githubusercontent.com/1534726/158492506-dae15881-e3f6-4829-9f53-2fe36e2a5eb5.png)

The results of `0.08333` are actually fairly significant and it might serve as a pretty decent default if you wanted tau to have a default value.

Reply:
I think for version 3.0 we can consider changing the default, for now it has to be `0` either way. I like that a higher tau gives more weight to more recent streaks. Yay graphs!!

<img width="940" alt="Screen Shot 2022-03-16 at 01 11 45+0000" src="https://user-images.githubusercontent.com/1247668/158497135-3318d81d-b198-4c8f-8f50-33532af91f4b.png">